contracts/liquidator/Liquidator.sol

```diff
@@ -29,14 +29,17 @@ interface IUniswapV2Router {
  */
 contract Liquidator is IUniswapV3FlashCallback, PeripheryImmutableState, PeripheryPayments {
     /** Errors */
+    error InsufficientAmountOut(address tokenIn, address tokenOut, uint256 amountIn, uint256 amountOut, uint256 amountOutMin, Exchange exchange, uint24 fee);
     error InsufficientBalance(uint256 available, uint256 required);
     error InvalidExchange();
+    error InvalidPoolConfig();
     error Unauthorized();
 
     /** Events **/
     event Absorb(address indexed initiator, address[] accounts);
+    event AbsorbedWithoutBuyingCollateral();
```

Comment:
nit: we've been using verbs for events so `AbsorbWithoutBuyingCollateral` would be more consistent

Reply:
good call. updated